package.sh

```diff
@@ -82,6 +83,19 @@ check_gopath() {
     echo "GOPATH ($GOPATH) looks sane, using $GOPATH_INSTALL for installation."
 }
 
+check_gvm() {
+    which gvm
+    if [ $? -ne 0 ]; then
+        echo "gvm not found -- aborting."
+        cleanup_exit $1
+    fi
+    gvm use $GO_VERSION
```

Comment:
This won't work because gvm script aren't inherited by sub-shells. You need to source ~/.gvm/scripts/gvm first to make it work